app/controllers/dashboard_controller.rb

```diff
@@ -1,9 +1,13 @@
 # frozen_string_literal: true
 
 require "velum/kubernetes"
+require "velum/suse_connect"
 
 # DashboardController shows the main page.
 class DashboardController < ApplicationController
+  skip_before_action :authenticate_user!, only: :autoyast
```

Comment:
Isn't this a bit dangerous ? The autoyast profile will contain sensitive data like the registration code. Isn't there a way to only serve this file to machines we know we can serve ? Just saying so things like the registration code don't leak into the outer world.